src/DOOM/m_menu.c

```diff
@@ -1418,6 +1418,23 @@ void M_FinishReadThis(int choice)
     M_SetupNextMenu(&MainDef);
 }
 
+//
+// When a secret is found (ColleagueRiley)
+//
+void doom_secretFound(int s) {
+    static lastTime = 0;
+    if (lastTime == 0) {
+        lastTime = I_GetTime();
+        S_StartSound(0, sfx_getpow);
+    }
+    
+    if (I_GetTime() >= lastTime + 20)
```

Comment:
Tried it and 20 frames (0.57 seconds) is really not long enough. Could we put it for like 50 frames (1.5 sec) maybe? What's the standard in other popular source ports that added this?

Reply:
Maybe. I can check the standard for gzdoom. The only concern about making it longer is that it seems to pause player movement if it's too long.